tests/test_peft.py

```diff
@@ -0,0 +1,49 @@
+import os
+import pytest
+import shutil
+
+from huggingface_hub import snapshot_download
+
+from petals.utils.peft import check_peft_repository, load_peft
+
+
+NOSAFE_PEFT_REPO = "timdettmers/guanaco-7b"
+SAFE_PEFT_REPO = "artek0chumak/guanaco-7b"
```

Comment:
Can you please replace this with a smaller pair of adapters? The current ones are too large to download in CI, this slows down tests by ~30%.

Reply:
Create smaller peft weights for `bloom-560m`